wled00/FX.cpp

```diff
@@ -3547,7 +3547,7 @@ uint16_t mode_exploding_fireworks(void)
   if (segs <= (strip.getMaxSegments() /4)) maxData *= 2; //ESP8266: 1024 if <= 4 segs ESP32: 2560 if <= 8 segs
   int maxSparks = maxData / sizeof(spark); //ESP8266: max. 21/42/85 sparks/seg, ESP32: max. 53/106/213 sparks/seg
 
-  unsigned numSparks = min(2 + ((rows*cols) >> 1), maxSparks);
+  unsigned numSparks = min(5 + ((rows*cols) >> 1), maxSparks);
```

Comment:
This change alone would be sufficient IMO. It will guarantee that minimum size is 5

Reply:
True. I do prefer to be absolutely explicit at the point of usage as well, though - it's saved me a couple times as I don't always remember the subtle relationships between static constants in future refactors.